zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -4995,6 +5053,44 @@ components:
           description: The version of the form.
           type: integer
           format: int64
+    AuthorizationSearchQueryRequest:
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryRequest"
+      type: object
+      properties:
+        filter:
+          allOf:
+            - $ref: "#/components/schemas/AuthorizationFilterRequest"
+    AuthorizationFilterRequest:
+      type: object
+      properties:
+        ownerType:
+          type: "string"
+    AuthorizationResponse:
+      type: "object"
+      properties:
+        ownerKey:
+          type: "string"
```

Comment:
according to the datamodel, ownerKey should be base64 integer

Reply:
Could you point me to data model please? as I checked in the code it's string down to AuthorizationEntity.

Reply:
I was looking at https://github.com/camunda/camunda/blob/main/webapps-schema/src/main/java/io/camunda/webapps/schema/entities/usermanagement/AuthorizationEntity.java, there is indeed inconsistency between the entity used by the exporter and the one used in the search